Use more meaningful label for output logging configuration

kenhys opened the pull request:
# Which issue(s) this PR fixes:

CSG#14

# What this PR does / why we need it:

Even though this option is disabled, it is still enabled for loggging with TRACE Log Monitor Window.
Thus this option actually controls whether output to Chronos trace.log or not.

It should represent actual behaviour precisely.

# How to verify the fixed issue:

1. Launch ChronosN.exe
2. Click Tools > Preferences menu
3. Check Logging category and DebugLog preference  

## Expected result:

* Label is changed to "Enable logging output to file"

kenhys committed:
Use more meaningful label for output logging configuration

Even though this option is disabled, it is still enabled for
loggging with TRACE Log Monitor Window.
Thus this option actually controls whether output to Chronos trace.log or not.

It should represent actual behaviour precisely.

Signed-off-by: Kentaro Hayashi <hayashi@clear-code.com>

kenhys commented:
![enable-logging-option-en](https://github.com/ThinBridge/Chronos/assets/225841/b086f25b-1f43-4efa-93e8-b20c69e53952)

![chronos-pr-125-ja](https://github.com/ThinBridge/Chronos/assets/225841/aadd031c-d9e0-459c-9d84-dd847d200371)

yashirot commented:
The improved text in Japanese is looking good to me.

As for text in English, however,
* 'Enable logging to file,' the shorten by omitting 'output,' might be simpler and better.
* Text within 'DebugLog' box is apparently not fully shown to the right end--This should be another issue anyway.

kenhys committed:
Fix hidden explanation of checkbox in english

The following label is hidden because of missing width.

  2:Output only general information [GE] and URL [URL] to a log file

HashidaTKS commented on Sazabi.rc:
```diff
@@ -1590,11 +1590,12 @@ END
 IDD_SETTINGS_DLG_LOG DIALOGEX 0, 0, 370, 291
 STYLE DS_SETFONT | DS_CONTROL | WS_CHILD | WS_CAPTION
-CAPTION "Logging"
+CAPTION "s"
```
Is this an automatically generated change?
Can we revert this CAPTION's change?
I think "Logging" is better.

kenhys replied:
This is unintended change, I'll fix it.

kenhys replied:
Fixed.

kenhys commented:
rebased with master.

HashidaTKS commented:
LGTM.

HashidaTKS commented:
Memo: 

> Text within 'DebugLog' box is apparently not fully shown to the right end--This should be another issue anyway.

This is also fixed at this PR.
